src/main/scala/no/ndla/articleapi/controller/ConceptController.scala

```diff
@@ -151,25 +190,27 @@ trait ConceptController {
       }
 
       result match {
-        case Success(searchResult) => searchResult
-        case Failure(ex)           => errorHandler(ex)
+        case Success(searchResult) =>
+          val responseHeader = searchResult.scrollId.map(i => this.scrollId.paramName -> i).toMap
+          Ok(searchConverterService.asApiConceptSearchResult(searchResult), headers = responseHeader)
```

Comment:
Det er ikkje fare for at du overskriver andre headers her?

Reply:
`apply` funksjonen til `Ok` defaulter til `Map()`, så vi overskriver ingenting.

scalatra/jetty legger på headere senere, så vi vil fortsatt få med de "vanlige" respons headerene:
```
Content-Length: 0
Date: Wed, 02 Jan 2019 11:28:17 GMT
Content-Type: application/json;charset=utf-8
Server: Jetty(9.4.12.v20180830)
search-context: DnF1ZXJ5VGhlbkZl...
```